client/disputes/index.tsx

```diff
@@ -99,55 +102,65 @@ const headers = [
 	},
 ];
 
-export const DisputesList = () => {
+export const DisputesList = (): JSX.Element => {
 	const { disputes, isLoading } = useDisputes( getQuery() );
 
+	const {
+		disputesSummary,
+		isLoading: isSummaryLoading,
```

Comment:
Aren't you overriding the previous `isLoading` variable here? I think we should keep hold of both API calls state separately.

Reply:
No, I don't think it is overridden in this case. The call to `useDisputesSummary` returns an object that has the `isLoading` property, but it is stored into `isSummaryLoading ` variable when the destructuring is done. We have two variables to indicate the loading state: `isLoading` for disputes list and `isSummaryLoading` for disputes summary.